src/media/js/previews-buttons.js

```diff
@@ -0,0 +1,59 @@
+/*
+    Arrow buttons for navigating back and forth in a preview tray that tell
+    Flipsnap what to do.
+
+    - Buttons disabled if there is no room to scroll left or right.
+    - We limit Flipsnap's max scroll position by 1 so we cant't scroll too far.
+    - If no room to scroll left or right (e.g., not enough previews), hide
+      buttons completely.
+*/
+define('previews-buttons', ['core/z'], function(z) {
```

Comment:
![dead-unicorn1](https://cloud.githubusercontent.com/assets/23885/6808097/cbd9aca6-d217-11e4-9aa3-4ff921481df4.jpg)

DEATH TO WHIMSY.